calico/tests/conftest.py

```diff
@@ -23,13 +23,11 @@
 
 def setup_calico():
     # Deploy calico
-    # Pinned the test manifests to v3.22. v3.23 deprecated a metric that we check for `felix_active_local_tags`
-    # https://projectcalico.docs.tigera.io/archive/v3.22/reference/felix/prometheus
-    run_command(["kubectl", "apply", "-f", "https://projectcalico.docs.tigera.io/archive/v3.22/manifests/calico.yaml"])
+    run_command(["kubectl", "apply", "-f", "https://projectcalico.docs.tigera.io/archive/v3.24/manifests/calico.yaml"])
 
     # Install calicoctl as a pod
     run_command(
-        ["kubectl", "apply", "-f", "https://projectcalico.docs.tigera.io/archive/v3.22/manifests/calicoctl.yaml"]
+        ["kubectl", "apply", "-f", "https://projectcalico.docs.tigera.io/archive/v3.24/manifests/calicoctl.yaml"]
```

Comment:
Would it be a good idea to host the manifest files in the repo in case they are removed one day?

Reply:
Sounds good!